### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by ampo0809**
- Modified access levels
- Updated default parameters

### CODE REVIEW

File: Sources/FSRS/Models/FSRSDefaults.swift [hunk 1]
```diff
@@ -9,11 +9,7 @@ import Foundation
 public class FSRSDefaults {
     var defaultRequestRetention = 0.9
     var defaultMaximumInterval = 36500.0
-    var defaultW = [
-      0.4072, 1.1829, 3.1262, 15.4722, 7.2102, 0.5316, 1.0651, 0.0234, 1.616,
-      0.1544, 1.0824, 1.9813, 0.0953, 0.2975, 2.2042, 0.2407, 2.9466, 0.5034,
-      0.6567,
-    ]
+    var defaultW = [0.40255, 1.18385, 3.173, 15.69105, 7.1949, 0.5345, 1.4604, 0.0046, 1.54575, 0.1192, 1.01925, 1.9395, 0.11, 0.29605, 2.2698, 0.2315, 2.9898, 0.51655, 0.6621]
```
Comment (ningkaiqiang) [inline Sources/FSRS/Models/FSRSDefaults.swift:12]: @L-M-Sherlock Please check the defaultW is right or not. Other changes look good to me.

**Review by ningkaiqiang — COMMENTED** [review]
The changes are good, but the access level seems needs more adjustment, I will update it later today.

File: Sources/FSRS/Models/FSRSDefaults.swift (hunk 1 above)
Comment (L-M-Sherlock) [reply Sources/FSRS/Models/FSRSDefaults.swift:12]: It's consistent with this:

https://github.com/open-spaced-repetition/fsrs-rs/blob/7477d2bf20749e393c437007564bd87519b7e70d/src/inference.rs#L25-L28

**Review by L-M-Sherlock — APPROVED** [review]

**Comment by L-M-Sherlock** [comment]
Oops. The unit tests are still based on the old parameters.

File: Sources/FSRS/Models/FSRSDefaults.swift (hunk 1 above)
Comment (ampo0809) [reply Sources/FSRS/Models/FSRSDefaults.swift:12]: I did copy them from that link. Sorry, forgot to pay an eye to to tests.

Comment (ningkaiqiang) [reply Sources/FSRS/Models/FSRSDefaults.swift:12]: This project currently is tracking ts-fsrs changes, the tests too, but this change is come from fsrs-rs, seems like a newer version default parameters, that way, we should update the tests as well. But there is other things need consider, I am not sure the algo is updated as well in fsrs-rs or not.
@L-M-Sherlock Do you have time to confirm this? 😛

Comment (L-M-Sherlock) [reply Sources/FSRS/Models/FSRSDefaults.swift:12]: I recommend reverting the change of default weights.

Comment (L-M-Sherlock) [inline Sources/FSRS/Models/FSRSDefaults.swift:12]:
```suggestion
    var defaultW = [0.4072, 1.1829, 3.1262, 15.4722, 7.2102, 0.5316, 1.0651, 0.0234, 1.616, 0.1544, 1.0824, 1.9813, 0.0953, 0.2975, 2.2042, 0.2407, 2.9466, 0.5034, 0.6567]
```

**Comment by L-M-Sherlock** [comment]
```
{"message":"Token required - not valid tokenless upload"}
```

cc @ningkaiqiang @ishiko732

**Comment by ishiko732** [comment]
> ```
> {"message":"Token required - not valid tokenless upload"}
> ```
> 
> cc @ningkaiqiang @ishiko732

In ts-fsrs, the following is used for uploading coverage reports to Codecov:

```yaml
- name: Upload coverage reports to Codecov
  uses: codecov/codecov-action@v4
  with:
    token: ${{ secrets.CODECOV_TOKEN }}
    fail_ci_if_error: true
```
- https://github.com/open-spaced-repetition/ts-fsrs/pull/107/commits/582785326453b7e90aa799a961c5ec21b94d3b9a
- https://github.com/open-spaced-repetition/ts-fsrs/blob/main/.github/workflows/ci.yml

**Comment by ningkaiqiang** [comment]
Is the token store in project settings expired or changed? @L-M-Sherlock @ishiko732

**Comment by ishiko732** [comment]
> Is the token store in project settings expired or changed? @L-M-Sherlock @ishiko732

I do not know what the token currently set for the repo is. Please @L-M-Sherlock check the token: https://app.codecov.io/gh/open-spaced-repetition/swift-fsrs/config/general

**Pull request merged**